std_msgs/Empty e
---
int32 x
int32 y
